swagger/v1/swagger.yaml

```diff
@@ -328,9 +328,9 @@ paths:
                       `post'"
                     - "/Users/colin.bruce/.rvm/gems/ruby-3.1.2/gems/actionpack-7.0.4/lib/action_dispatch/testing/integration.rb:370:in
                       `post'"
-                    - "/Users/colin.bruce/.rvm/gems/ruby-3.1.2/gems/rswag-specs-2.6.0/lib/rswag/specs/example_helpers.rb:20:in
+                    - "/Users/colin.bruce/.rvm/gems/ruby-3.1.2/gems/rswag-specs-2.7.0/lib/rswag/specs/example_helpers.rb:20:in
```

Comment:
**not a blocker** as i can see they are already in and have no idea how to prevent this, but be good to, somehow 🤷‍♂️,  exclude these. Would a  swaggerize on my machine  overwrite these?

Reply:
https://github.com/ministryofjustice/legal-framework-api/pull/771 😀

Reply:
just saw and approved that one after wiring this 😄